packages/core/src/utils/useDuration.ts

```diff
@@ -0,0 +1,29 @@
+import {useScene} from './useScene';
+
+/**
+ * Register a time event and get its duration in seconds.
+ *
+ * @example
+ * This can be used to better specify when an animation should start
```

Comment:
This should be under `@remarks`
Examples are collapsed by default when reading the docs

Reply:
Do you mean to replace `@example` with `@remarks` or add a `@remarks` above `@example`?

Reply:
Add remarks above example

Reply:
Hope it's now how you meant it

Reply:
Yup, it's okay